src/locale/locales/pt-BR/messages.po

```diff
@@ -214,7 +214,7 @@ msgstr "<0>{0}, </0><1>{1}, </1>e {2, plural, one {# outro} other {# outros}} es
 
 #: src/screens/StarterPack/Wizard/index.tsx:497
 #~ msgid "<0>{0}, </0><1>{1}, </1>and {2} {3, plural, one {other} other {others}} are included in your starter pack"
-#~ msgstr ""
+#~ msgstr  "<0>{0}, </0><1>{1}, </1>e {2} {3, plural, one {outro} other {outros}} estão incluídos no seu kit inicial"
```

Comment:
In this PR https://github.com/bluesky-social/social-app/pull/5090, @Garccez  translated kit as "pacote", I think it's interesting to continue with this approach to maintain coherence.

Reply:
```suggestion
#~ msgstr  "<0>{0}, </0><1>{1}, </1>e {2} {3, plural, one {outro} other {outros}} estão incluídos no seu pacote inicial"
```

Reply:
According to the last PR it was defined that starter pack would be translated to 'kit inicial'.